test/plugins/collections.test.js

```diff
@@ -368,4 +368,60 @@ describe('collection plugin test', () => {
             });
         });
     });
+
+    describe('DELETE /collections/{id}', () => {
+        const id = testCollection.id;
+        let options;
+
+        beforeEach(() => {
+            options = {
+                method: 'DELETE',
+                url: `/collections/${id}`,
+                credentials: {
+                    username,
+                    scope: ['user']
+                }
+            };
+
+            collectionFactoryMock.get.withArgs(id).resolves(collectionMock);
+        });
+
+        it('returns 204 when delete is successful', () =>
+            server.inject(options).then((reply) => {
+                assert.equal(reply.statusCode, 204);
+                assert.calledOnce(collectionMock.remove);
+            })
+        );
+
+        it('returns 401 when user does not have permiddion', () => {
```

Comment:
This is weird... I fixed that typo before and pushed it earlier but it's not updating on here? It's permission on my local